Header header
string algo
string command
string speed 
int8 abs_speed
int8 precedence
geometry_msgs/Twist cmd_custom